### PR TITLE
Fix english typos in dart files

### PR DESCRIPTION
**Pull request by orchardbirds**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x ] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [ x] Tested your changes, especially after any code review iterations.
- [ x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [ x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x ] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

After all boxes above are checked, request and receive an Approved review from any team member knowledgable in the area (TODO team member list).  Once approved, the team member will assign your review to a Committer or use the `needs-merge` label.

## What does this PR accomplish?

 Makes dart english dart files more easily readable

It also takes into consideration jpelgrim's fix. Apologies for the double-work here and  apologies to @jpelgrim

Closes #404 and #410 
## Did you add any dependencies?
No
## How did you test the change?
Compiled app

### CODE REVIEW

File: client/flutter/lib/generated/intl/messages_en.dart [hunk 1]
```diff
@@ -23,12 +23,12 @@ class MessageLookup extends MessageLookupByLibrary {
...
     "childRoutines" : MessageLookupByLibrary.simpleMessage("Keep to regular routines and schedules as much as possible, or help create new ones in a new environment, including school/learning as well as time for safely playing and relaxing."),
-    "childStress" : MessageLookupByLibrary.simpleMessage("Children may respond to stress in different ways such as being more clingy, anxious, withdraw, angry or agitated, bedwetting etc\nRespond to your child\'s reactions in a supportive way, listen to their concerns and give them extra love and attention."),
-    "cougningAndSneezing" : MessageLookupByLibrary.simpleMessage("When coughing and sneezing cover mouth and nose with flexed elbow or tissue"),
+    "childStress" : MessageLookupByLibrary.simpleMessage("Children may respond to stress in different ways such as being more clingy, anxious, withdrawn, angry or agitated; by bed-wetting etc\nRespond to your child\'s reactions in a supportive way, listen to their concerns and give them extra love and attention."),
```
Comment (epicfaace) [inline client/flutter/lib/generated/intl/messages_en.dart]: I think we can leave it "bedwetting" (https://books.google.com/ngrams/graph?content=bedwetting%2C+bed-wetting&year_start=1800&year_end=2008&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cbedwetting%3B%2Cc0%3B.t1%3B%2Cbed%20-%20wetting%3B%2Cc0#t1%3B%2Cbedwetting%3B%2Cc0%3B.t1%3B%2Cbed%20-%20wetting%3B%2Cc0)

```suggestion
    "childStress" : MessageLookupByLibrary.simpleMessage("Children may respond to stress in different ways such as being more clingy, anxious, withdrawn, angry or agitated; by bedwetting etc\nRespond to your child\'s reactions in a supportive way, listen to their concerns and give them extra love and attention."),
```

File: client/flutter/lib/l10n/intl_en.arb [hunk 2]
```diff
@@ -7,19 +7,19 @@
     "shareTheApp": "Share the App",
     "healthCheck": "Health Check",
     "feelingDistressed": "Feeling Distressed?",
-    "washHands": "Wash your hand often with soap and running water frequently",
-    "cougningAndSneezing": "When coughing and sneezing cover mouth and nose with flexed elbow or tissue",
-    "throwAwayTissue":"Throw tissue into closed bin immediately after use",
+    "washHands": "Wash your hands often with soap and running water frequently",
```
Comment (SamMousa) [inline client/flutter/lib/l10n/intl_en.arb]: Having both often & frequently doesn't make sense to me. Should we rewrite that?

**Comment by jpelgrim** [comment]
No worries @orchardbirds!

File: client/flutter/lib/generated/intl/messages_en.dart (hunk 1 above)
Comment (orchardbirds) [reply client/flutter/lib/generated/intl/messages_en.dart]: thanks!

File: client/flutter/lib/l10n/intl_en.arb (hunk 2 above)
Comment (orchardbirds) [reply client/flutter/lib/l10n/intl_en.arb]: good point, it is there twice. I think once is enough - will change

Comment (orchardbirds) [reply client/flutter/lib/l10n/intl_en.arb]: Oh I misunderstood originally and removed the 'washHandsFrequently' argument - is that OK or too much?

File: client/flutter/lib/generated/intl/messages_ar.dart [hunk 3]
```diff
@@ -40,7 +40,6 @@ class MessageLookup extends MessageLookupByLibrary {
     "socialDistancing" : MessageLookupByLibrary.simpleMessage(""),
     "throwAwayTissue" : MessageLookupByLibrary.simpleMessage(""),
     "washHands" : MessageLookupByLibrary.simpleMessage(""),
-    "washHandsFrequently" : MessageLookupByLibrary.simpleMessage(""),
```
Comment (epicfaace) [inline client/flutter/lib/generated/intl/messages_ar.dart]: Did you mean to remove this?

Comment (orchardbirds) [reply client/flutter/lib/generated/intl/messages_ar.dart]: Yeah, do we need washHands and washHandsFrequently? Also, see above: it was a misunderstanding from another comment haha. I can put it back if needed

File: client/flutter/lib/l10n/intl_en.arb (hunk 2 above)
Comment (epicfaace) [reply client/flutter/lib/l10n/intl_en.arb]: I think he meant just the English wording.

Comment (orchardbirds) [reply client/flutter/lib/l10n/intl_en.arb]: agreed, will put it back - don't want to change too much in 1 MR

File: client/flutter/lib/generated/intl/messages_en.dart (hunk 1 above)
Comment (orchardbirds) [reply client/flutter/lib/generated/intl/messages_en.dart]: undone

**Pull request merged**